src/content/en/2020/seo.md

```diff
@@ -569,7 +568,7 @@ Feature | Mobile | Desktop
 `orientation` | 33.48% | 33.49%
 `max-device-width` | 26.23% | 28.15%
 
-<figcaption>{{ figure_link(caption="Media query usage.", sheets_gid="1141218471", sql_file="TODO.sql") }}</figcaption>
+<figcaption>{{ figure_link(caption="Media query usage.", sheets_gid="1141218471", sql_file="TODO..sql") }}</figcaption>
```

Comment:
@Tiggerito can you suggest the correct SQL file for this figure?

Reply:
It looks like I stole this from you 😮

I asked the CSS chapter for some help:

https://github.com/HTTPArchive/almanac.httparchive.org/issues/898#issuecomment-706792639

And you provided the data...

https://docs.google.com/spreadsheets/d/1sMWXWjMujqfAREYxNbG_t1fOJKYCA6ASLwtz4pBQVTw/edit#gid=1374950017

Reply:
@rviscomi what's the solution here. Is there any formal SQL to reference?